### PR TITLE
Add documentation about Secret Service

### PR DESCRIPTION
**Pull request by StackScribe**
https://github.com/keptn/keptn.github.io/issues/1003 

Move content from https://github.com/keptn/keptn/blob/master/statistics-service/README.md into Concepts section of documentation

### CODE REVIEW

File: content/docs/concepts/secrets/index.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+title: Secret Service
+description: Learn how to manage secrets in a Keptn cluster.
+weight: 20
+keywords: [keptn, use-cases]
+---
+
+The **Secret Service** is used to manage secrets in a Keptn Cluster.
+It provides a simple API for creating, updating or deleting secrets in a specific secret backend such as Kubernetes or vault.
+
+**NOTE:** The current implementation only supports Kubernetes as a secret backend.
```
Comment (oleg-nenashev) [inline content/docs/concepts/secrets/index.md:11]:
```suggestion
**NOTE:** The current implementation only supports [Kubernetes Secrets](https://kubernetes.io/docs/concepts/configuration/secret/) as a secret backend.
```

File: content/docs/concepts/secrets/index.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
+---
+title: Secret Service
+description: Learn how to manage secrets in a Keptn cluster.
+weight: 20
+keywords: [keptn, use-cases]
+---
+
+The **Secret Service** is used to manage secrets in a Keptn Cluster.
```
Comment (oleg-nenashev) [inline content/docs/concepts/secrets/index.md:8]: Add version where it was introduced?

File: content/docs/concepts/secrets/index.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+## Generate  Swagger doc from source
+
+1. Download and install Swag for Go by calling the following in a fresh terminal:
+
+```
+go get -u github.com/swaggo/swag/cmd/swag
+```
+2. `cd` to the SecretService's root folder
+3. Run `swag init`
```
Comment (oleg-nenashev) [inline content/docs/concepts/secrets/index.md:59]: This is developer documentation, not the user one. 

```suggestion
```

**Review by oleg-nenashev — CHANGES_REQUESTED** [review]
Ideally we should publish the API documentation on the website or on SwaggerHub. I can help with automation. CC @mowies who might recommend the right way

File: content/docs/concepts/secrets/index.md (hunk 2 above)
Comment (StackScribe) [reply content/docs/concepts/secrets/index.md:8]: Excellent idea!  How do I find that information?

Comment (oleg-nenashev) [reply content/docs/concepts/secrets/index.md:8]: 0.8.1 according to https://github.com/keptn/keptn/pull/3595

**Comment by mowies** [comment]
Hey @StackScribe !
You still have a failing DCO check. If you go to the checks tab in the PR, the DCO check will give you helpful instructions on how to fix the failing checks.

**Review by oleg-nenashev — APPROVED** [review]
Let's merge it so that we can move forward with docs

**Pull request merged**